FlightPlanner.Web/FlightPlanner.Web/Storage/FlightStorage.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FlightPlanner.Web.DbContext;
+using FlightPlanner.Web.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace FlightPlanner.Web.Storage
+{
+    public class FlightStorage
+    {
+        private static PageResult _pageResults = new PageResult();
+        private static readonly object flightLock = new object();
+        private static int _id;
+
+        public static Flight GetById(int id, FlightPlannerDbContext _context)
+        {
+            lock (flightLock)
+            {
+                return _context.Flights.Include(f => f.From).Include(f => f.To).FirstOrDefault(f => f.Id == id);
+            }
+        }
+
+        public static void ClearFlights(FlightPlannerDbContext _context)
+        {
+            foreach (Airport airport in _context.Airports)
+                _context.Airports.Remove(airport);
+
+            foreach (Flight flight in _context.Flights)
+                _context.Flights.Remove(flight);
+
+            _context.SaveChanges();
+        }
+
+        public static Flight AddFlight(Flight flight, FlightPlannerDbContext _context)
+        {
+            lock (flightLock)
+            {
+
+                _context.Flights.Add(flight);
+                _context.SaveChanges();
+
+
+                return flight;
+            }
+        }
+
+        public static bool NotAbleToAddSameFlight(Flight flight, FlightPlannerDbContext _context)
+        {
+            lock (flightLock)
+            {
+                return _context.Flights.Include(f => f.From)
+                    .Include(f => f.To)
+                    .Any(f => f.ArrivalTime == flight.ArrivalTime
+                              && f.DepartureTime == flight.DepartureTime
+                              && f.Carrier == flight.Carrier
+                              && f.From.AirportCode == flight.From.AirportCode
+                              && f.To.AirportCode == flight.To.AirportCode);
+            }
+        }
+
+        public static bool NotAbleToAcceptWrongValues(Flight flight)
```

Comment:
šis un 2 nākošo metožu nosaukumi ir vairāk statement uz kuru nevar atbildēt ar jā vai nē. :) 
labāk būtu piemēram FlightHasWrongValues vai FlightHasInvalidValues uz šādu var atbildēt ar jā vai nē.. 
un līdz ar ko uzlabojas lasāmība kodam.. :) if flight has wrong values return bad request.. etc. :)